manifests/s/SmartSoft/SmartFTP/10.0.2950.0/SmartSoft.SmartFTP.yaml

```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.1.0.schema.json
+
+PackageIdentifier: SmartSoft.SmartFTP
+PackageVersion: 10.0.2950.0
+PackageLocale: en-US
+Publisher: SmartSoft Ltd.
+PublisherUrl: https://www.smartftp.com
+PublisherSupportUrl: https://www.smartftp.com/support
+Author: SmartSoft Ltd.
+PackageName: SmartFTP Client
+License: Commercial
+LicenseUrl: https://www.smartftp.com/static/Products/Client/License.txt
+ShortDescription: SmartFTP is a fast and reliable FTP, FTPS, SFTP, HTTP, Amazon S3, WebDAV, Google Drive, Google Cloud Storage, Microsoft OneDrive, Backblaze B2, Telnet, Terminal client with lots of features and a modern user interface.
+Moniker: smartftp
+Tags:
+- ftp
+- sftp
+- webdav
+- onedrive
+- gdrive
+- ssh
+ReleaseNotesUrl: https://www.smartftp.com/en-us/changelog/1
+MinimumOSVersion: 10.0.14393.0
+Installers:
+- InstallerLocale: en-US
+  Platform:
+  - Windows.Desktop
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://static.smartftp.com/static/Products/Client/10.0.2950.0/x64/SmartFTP.msi
+  InstallerSha256: 2DFF4AEC45EAE5F3D7A679A56436DAE0F7890CD4BEC2A4F901D311A2912301B8
+  ProductCode: '{75A4624C-05C3-4709-998F-20EA62ED2774}'
+ManifestType: singleton

```

Comment:
Singleton types are not preferred. If possible, can you update this PR to a multi-manifest? 

https://github.com/microsoft/winget-pkgs/blob/6e033783b539658fe919e503e4a47894e63b1888/doc/manifest/schema/1.1.0/singleton.md?plain=1#L19

Reply:
Not for this PR, but we will no longer submit singleton manifests in future commits.